<launch>
    <!-- XSENS to KINOVA -->
    <node name="xsens2kinova"  pkg="get_hand_position_on_kinova_coord" type="get_hand_position_on_kinova_coord_node" output="screen" />

    <!--Main Program-->
    <!-- <node pkg="kinova_test" type="adaptive_control" name="adaptive_control_node" output="screen" /> -->
</launch>